### ISSUE TITLE
AppSpector iOS Crash on v1.4.5

### ISSUE THREAD

**Issue by luthfifr** [appspector/ios-sdk#23]
I'm experiencing crash on app start up when running the project directly from XCode (debug configuration),  due to AppSpector. Here's more detail about it:

Installed version: v1.4.5 via Cocoapods

Running device: iPhone 6s
iOS version: 14.3

Crash log:
<img width="1792" alt="Screen Shot 2022-01-04 at 11 18 30 AM" src="https://user-images.githubusercontent.com/22187764/148009345-c59462e3-bd09-4008-9bf3-0029139f89a4.png">

```
*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]
(
	0   CoreFoundation                      0x0000000185e759ec 6FD5304A-9759-3E22-A863-456EAB55F4CC + 1157612
	1   libobjc.A.dylib                     0x000000019a1deb54 objc_exception_throw + 56
	2   CoreFoundation                      0x0000000185edfd98 6FD5304A-9759-3E22-A863-456EAB55F4CC + 1592728
	3   CoreFoundation                      0x0000000185eeb1f4 6FD5304A-9759-3E22-A863-456EAB55F4CC + 1638900
	4   CoreFoundation                      0x0000000185d708a8 6FD5304A-9759-3E22-A863-456EAB55F4CC + 88232
	5   CoreFoundation                      0x0000000185d63c04 6FD5304A-9759-3E22-A863-456EAB55F4CC + 35844
	6   AppSpectorSDK                       0x00000001065cb6bc -[ASSessionStarter errorFromAPIError:] + 124
	7   AppSpectorSDK                       0x00000001065ca938 __55-[ASSessionStarter startSessionWithRequest:completion:]_block_invoke + 260
	8   AppSpectorSDK                       0x00000001065403a0 __85+[ASNetworkObserver asyncCompletionWrapperForOriginalRequestID:mechanism:completion:]_block_invoke + 448
	9   NewRelic                            0x0000000109d1bd34 NRMAOverride__uploadTaskWithStreamedRequest + 880
	10  CFNetwork                           0x000000018646b8e4 CFNetServiceBrowserSearchForServices + 75936
	11  CFNetwork                           0x000000018647c3b4 _CFHTTPMessageSetResponseProxyURL + 9332
	12  libdispatch.dylib                   0x0000000110813ce4 _dispatch_call_block_and_release + 24
	13  libdispatch.dylib                   0x0000000110815528 _dispatch_client_callout + 16
	14  libdispatch.dylib                   0x000000011081c5ac _dispatch_lane_serial_drain + 748
	15  libdispatch.dylib                   0x000000011081d268 _dispatch_lane_invoke + 504
	16  libdispatch.dylib                   0x0000000110828a5c _dispatch_workloop_worker_thread + 1456
	17  libsystem_pthread.dylib             0x00000001ce56f5a4 _pthread_wqthread + 272
	18  libsystem_pthread.dylib             0x00000001ce572874 start_wqthread + 8
```

Anyone can help?
Thanks.

---
P.S.: Can anyone tell me what is the last stable version?

**Comment by deszip** [comment]
Hi @luthfifr, thanks for the detailed report. Working on this, will be fixed in the next release (1.4.6).
Current stable version (1.4.5 at the moment) is always the latest version on this repo, you can check it in the releases section.

**Comment by deszip** [comment]
Version 1.4.6 is out and has a potential fix for this issue. Closing this, feel free to reopen if issue still persists.
